frida_tools/tracer.py

```diff
@@ -54,6 +54,8 @@ def process_builder_arg(option, opt_str, value, parser, method, **kwargs):
                     type='string', action='callback', callback=process_builder_arg, callback_args=(pb.include_debug_symbol,))
             parser.add_option("-q", "--quiet", help="do not format output messages", action='store_true', default=False)
             parser.add_option("-d", "--decorate", help="Add module name to generated onEnter log statement", action='store_true', default=False)
+            parser.add_option("-S", "--init-session", help="Path to JavaScript file used to initialize the session", metavar="PATH",
```

Comment:
```suggestion
            parser.add_option("-S", "--init-session", help="path to JavaScript file used to initialize the session", metavar="PATH",
```

(For consistency with the other help messages. I totally missed this on the `--decorate` PR.)

Reply:
Both --init-session and --decorate help strings are now lowercase.